grc20-core/src/mapping/entity/find_path.rs

```diff
@@ -0,0 +1,163 @@
+use neo4rs::Path;
+
+use crate::{
+    entity::{EntityFilter, EntityNode},
+    error::DatabaseError,
+    mapping::{
+        order_by::FieldOrderBy,
+        query_utils::{
+            query_builder::{MatchQuery, QueryBuilder, Subquery},
+            VersionFilter,
+        },
+        AttributeFilter, PropFilter, Query,
+    },
+    system_ids::SCHEMA_TYPE,
+};
+
+pub struct Relation {
+    pub nodes_ids: Vec<String>,
+    pub relations_ids: Vec<String>,
+}
+
+pub struct FindPathQuery {
+    neo4j: neo4rs::Graph,
+    id1: String,
+    id2: String,
+    filter: EntityFilter,
+    order_by: Option<FieldOrderBy>,
+    limit: usize,
+    skip: Option<usize>,
+    space_id: Option<PropFilter<String>>,
+    version: VersionFilter,
+    _phantom: std::marker::PhantomData<EntityNode>,
```

Comment:
Since `FindPathQuery` does not have a generic type parameter anymore, we don't need to have a `_phantom` field!